pages/integrations/sso.md.erb

```diff
@@ -54,6 +54,26 @@ If you are the administrator of an organization within Buildkite with an existin
 
 If you'd like to have some help with the migration, contact support@buildkite.com.
 
+## SSO Session Duration
+
+You can configure the SSO Session Duration to timeout after a predetermined time. When the specified duration elapses, the user will be signed out of the session.
+
+To set the Session Duration you can either use the [GraphQL API](/docs/apis/graphql/graphql-cookbook#update-the-default-sso-provider-session-duration) or complete
```

Comment:
Yes, linking it to the cookbook would be handy (I wasn't aware we have this content already). Thanks for adding this in.